build/integration.sh

```diff
@@ -26,7 +26,8 @@ printf "" # Refresh sudo credentials if necessary.
 function start {
   set +e  # We want to handle errors if cAdvisor crashes.
   echo ">> starting cAdvisor locally"
-  GORACE="halt_on_error=1" ./cadvisor --docker_env_metadata_whitelist=TEST_VAR --v=6 --logtostderr $CADVISOR_ARGS &> "$log_file"
+  # This cpu, cpuset, percpu, app, memory, disk, diskIO, network, perf_event metrics should be enabled.
```

Comment:
This -> The ?

Btw.  If this PR would depend on PR#2902, it would be enough to change "disable_metrics" for "enable_metrics" option, to document what metrics the testing actually needs.